README.md

```diff
@@ -125,7 +125,7 @@ Agents facilitate the interaction from clients with canisters on the Internet Co
 
 ### Motoko
 
-- [Motoko](#Motoko) - Simple high-level language for writing canisters.
+- [Motoko](https://github.com/dfinity/motoko) - Simple high-level language for writing canisters.
```

Comment:
I would prefer to change it to #motoko-1 so that it can link to the previous Motoko section, which already provides a link to the GitHub repo
```suggestion
- [Motoko](#motoko-1) - Simple high-level language for writing canisters.
```